pkg/scheduler/core/util.go

```diff
@@ -15,8 +15,29 @@ import (
 	"github.com/karmada-io/karmada/pkg/util"
 )
 
+const (
+	NoClusterAvailableReason = "NoClusterAvailable"
+	NoClusterFitReason       = "NoClusterFit"
```

Comment:
Why not put these in file `pkg/scheduler/scheduler.go` L59?

Reply:
If placed in file `pkg/scheduler/scheduler.go`, package `scheduler` will be imported in package `core`, which is a circular dependency.

Reply:
I prefer to keep this information in the same place. So how about moving the related variable in `pkg/scheduler/scheduler.go` here?

Reply:
I will find a solution.